Paintroid/src/main/java/org/catrobat/paintroid/FileIO.java

```diff
@@ -85,28 +104,34 @@ public static Uri saveBitmapToFile(String fileName, Bitmap bitmap, ContentResolv
 		if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.Q) {
 			ContentValues contentValues = new ContentValues();
 			contentValues.put(MediaStore.Images.Media.DISPLAY_NAME, fileName);
-			contentValues.put(MediaStore.Images.Media.MIME_TYPE, "image/png");
+			contentValues.put(MediaStore.Images.Media.MIME_TYPE, "image/*");
```

Comment:
If i try to edit a picture from catroid and export it with default name (e.g. image2, format="jpg") the resulting file is a .bin file with mimetype application/octet-stream. I'm not sure if this happens due to this line but i'd suggest to distinguish between png/jpeg and set the corresponding mimetype rather than image/*